nearlib/test/integration.test.js

```diff
@@ -21,6 +25,73 @@ test('test creating default config', async () => {
     Near.createDefaultConfig();
 });
 
+describe("dev connect", () => {
+    let deps;
+    beforeEach(async () => {
+        const keyStore = new InMemoryKeyStore();
+        const storage = createFakestorage();   
+        deps = {
+            keyStore,
+            storage,
+            createAccount: (async (newAccountName, newAccountPublicKey) => {
+                const createAccountResponse = await account.createAccount(newAccountName, newAccountPublicKey, 1, aliceAccountName);
+                await nearjs.waitForTransactionResult(createAccountResponse);
+            })
+        };
+    });
+
+    test('test dev connect with no account creates a new account', async () => {
+        await dev.connect({deps});
+        expect(Object.keys(deps.keyStore.keys).length).toEqual(1);
+        const newAccountId = Object.keys(deps.keyStore.keys)[0];
+        const viewAccountResponse = await account.viewAccount(newAccountId);
+        const newAccountKeyPair = await deps.keyStore.getKey(newAccountId);
+        expect(newAccountKeyPair).toBeTruthy();
+        const expectedAccount = {
+            nonce: 0,
+            account_id: newAccountId,
+            amount: 1,
+            code_hash: newAccountCodeHash,
+            stake: 0,
+        };
+        expect(viewAccountResponse).toEqual(expectedAccount);
```

Comment:
You likely can simplify this a lot by using `https://jestjs.io/docs/en/expect#tomatchobjectobject`, cause stuff like `code_hash` doesn't seem crucial for these tests.

Reply:
sure, what's the priority of this vs getting the better api for the weekend though?

Reply:
Let's verify if fiddles work fine with `nearlib` at `b1fa4e3` (i.e. 0.3.0) and if it works fine we'll have the option to push as is.

Reply:
Will test f4c5363ee3d1142c8f4111ef9be48411548aff34 (same as b1fa4e3 + update dist file)

Reply:
works with fiddles on nearstudio that's live right now